pkg/controller/directvolumemigration/rsync.go

```diff
@@ -886,9 +966,17 @@ func (t *Task) createRsyncClientPods() error {
 					Requests: requests,
 				},
 			})
+			//var fsGroup *int64
```

Comment:
Left over testing code? otherwise can you explain in the comment some more?

Reply:
left over needs to be gone!!